Src/Witsml/Data/WitsmlRisk.cs

```diff
@@ -30,20 +30,10 @@ public class WitsmlRisk
         public string DTimStart { get; set; }
         [XmlElement("dTimEnd")]
         public string DTimEnd { get; set; }
-        [XmlElement("mdHoleStart")]
```

Comment:
There is no reason to delete these as they would have to be put back in in case of implementing copy operation for risk